src/presentation/options/options.js

```diff
@@ -1,35 +1,75 @@
 import WindowTitler from '/src/WindowTitler.js';
 import ProfileTitleRepository from '/src/persistence/ProfileTitleRepository.js';
+import UserWindowTitleRepository from '/src/persistence/UserWindowTitleRepository.js';
 import DefaultValues from '/src/model/DefaultValues.js';
 
 const windowTitler = new WindowTitler();
 const profileTitleRepository = new ProfileTitleRepository();
+const userWindowTitleRepository = new UserWindowTitleRepository();
 
-function setOptions(profileTitle, profileTitleSeparator) {
+
+// Profile Functions
+function setProfileOptions(profileTitle, profileTitleSeparator) {
   document.querySelector('#profile-title').value = profileTitle;
   document.querySelector('#profile-title-separator').value = profileTitleSeparator;
 }
 
-async function restoreOptions() {
+async function restoreProfileOptions() {
   const profileTitle = await profileTitleRepository.getProfileTitle();
   const profileTitleSeparator = await profileTitleRepository.getProfileTitleSeparator();
 
-  setOptions(profileTitle, profileTitleSeparator);
+  setProfileOptions(profileTitle, profileTitleSeparator);
 }
 
-function saveOptions(e) {
-  e.preventDefault();
-
+function saveProfileOptions() {
   const profileTitle = document.querySelector('#profile-title').value;
   const profileTitleSeparator = document.querySelector('#profile-title-separator').value;
 
   windowTitler.saveProfileTitleAndRefreshPresentation(profileTitle, profileTitleSeparator);
 }
 
-function restoreDefaults() {
-  setOptions(DefaultValues.profileTitle, DefaultValues.profileTitleSeparator);
+function restoreProfileDefaults() {
+  setProfileOptions(DefaultValues.profileTitle, DefaultValues.profileTitleSeparator);
+}
+
+// Profile Actions
+restoreProfileOptions();
+document.querySelector('#save-profile').addEventListener('click', saveProfileOptions);
+document.querySelector('#restore-profile-defaults').addEventListener('click', restoreProfileDefaults);
+
+
+// Tags Functions
+function setWindowTitleTagsOptions(windowTitleOpeningTag, windowTitleClosingTag) {
+  document.querySelector('#window-title-opening-tag').value = windowTitleOpeningTag;
+  document.querySelector('#window-title-closing-tag').value = windowTitleClosingTag;
+}
+
+async function restoreWindowTitleTagsOptions() {
```

Comment:
restoreTagOptions, instead of restoreTagsOptions. 'tag' is an adjective here.

Reply:
Ok